examples/dynamic-share.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+// Load dependent packages and env data
+
+require_once __DIR__ . '/bootstrap.php';
+
+use Yoti\ShareUrl\DynamicScenarioBuilder;
+use Yoti\ShareUrl\Extension\LocationConstraintExtensionBuilder;
+use Yoti\ShareUrl\Policy\DynamicPolicyBuilder;
+
+$yotiClient = new Yoti\YotiClient(getenv('YOTI_SDK_ID'), getenv('YOTI_KEY_FILE_PATH'));
+
+$locationConstraint = (new LocationConstraintExtensionBuilder())
+    ->withLatitude(50.8169)
+    ->withLongitude(-0.1367)
+    ->withRadius(100000)
+    ->build();
+
+$policy = (new DynamicPolicyBuilder())
+    ->withFullName()
+    ->withDocumentDetails()
+    ->withDocumentImages()
+    ->withAgeOver(18)
+    ->withSelfie()
+    ->build();
+
+$scenario = (new DynamicScenarioBuilder())
+    ->withPolicy($policy)
+    ->withCallbackEndpoint('/profile.php')
+    ->withExtension($locationConstraint)
+    ->build();
+
+$shareUrlResult = $yotiClient->createShareUrl($scenario);
+?>
+<!DOCTYPE html>
+<html class="yoti-html">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Yoti Dynamic Share example</title>
+        <link rel="stylesheet" type="text/css" href="assets/css/index.css">
+        <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
+    </head>
+    <body class="yoti-body">
+        <main>
+            <section class="yoti-top-section">
+                <div class="yoti-logo-section">
+                    <a href="https://www.yoti.com" target="_blank">
+                        <img class="yoti-logo-image" src="assets/images/logo.png" srcset="assets/images/logo@2x.png 2x" alt="Yoti"/>
+                    </a>
+                </div>
+
+                <h1 class="yoti-top-header">We now accept Yoti</h1>
```

Comment:
Could we change this to have "Dynamic Share Example" too? Just to distinguish it from the standard index page more (I know the page title is also different)
```suggestion
                <h1 class="yoti-top-header">Dynamic Share Example</h1>
```